.travis.yml

```diff
@@ -0,0 +1,53 @@
+os:
+  - linux
+
+dist: bionic
+
+env:
+  matrix:
+    # cmake test
+    - TASK=cmake_test
+
+before_install:
+  # Configure Intel and Microsoft APT repositories
+  - sudo apt-get -q update
```

Comment:
Travis CI's [docs](https://docs.travis-ci.com/user/installing-dependencies) suggest not to run `apt-get upgrade` as it'll increase build times significantly:

"Do not run apt-get upgrade in your build as it downloads up to 500MB of packages and significantly extends your build time. Additionally, some packages may fail to update, which will lead to a failed build."

Is there any way around doing this for us?

Reply:
Oh we do an update, not an upgrade.

Reply:
Oops. My bad.